media/js/firefox/hello/index.js

```diff
@@ -119,12 +119,38 @@
         }
         // Hello exists in desktop version 35 and up
         else if (w.getFirefoxMasterVersion() >= 35) {
-            showFxFooterMessaging();
+            Promise.all([
```

Comment:
Even though this is behind a piece of UA detection, I guess we should still add a feature check too just to be on the safe side?